app/src/js/components/SortableTable/SortableTable.js

```diff
@@ -173,6 +175,7 @@ const SortableTable = ({
   useEffect(() => {
     if (typeof changeSortProps === 'function') {
       changeSortProps(sortBy);
+      dispatch(sortPersist(true, action));
```

Comment:
Once we have the action imported, this should look something like this

```suggestion
      dispatch(sortPersist(sortBy));
```